### DISCUSSION TITLE
Side browser mode like slidepad.app

### DISCUSSION THREAD

**Discussion by mr-cheffy** [zen-browser/desktop#812] [Ideas]
### What feature would you like to see?

It would be great if there was sidebrowser mode.
The example for this would be the slidepad.app.
Hope something like this can be implemented.
Thanks

### What platform are you seeing the problem on?

Windows

 
 ~ _Created automatically from issue: https://github.com/zen-browser/desktop/issues/518_

**Comment by rp1231** [comment]
Maybe just an optional keyboard shorcut field can be introduced to hide/show the browser.
That would be enough to emulate the slidepad mode.
And an option to move the browser to the windows tray area and minimize/close to tray in compact mode.

**Comment by KiKaraage** [comment]
cc @rp1231 - I think the Zen Sidebar/Webpanel features already do this job quite well. Or, if you want to use it externally, without opening the whole browser, the Edge sidebar should've got you covered.

I don't know. I feel like your concern is very valid since I myself love these kind of feature too, but Zen is a small team, they have so many feature parity with other browser before they can get out of beta, and I don't think this feature is good enough to be prioritized by the Zen team. And I think a sidebrowser mode would be too bloated for this browser, similar to how many people/enthusiasts feel about Edge.

1. So, I'd rather wait until Zen is "completed" (moving to beta or regular phase) first before requesting this feature.
2. Or, I'd rather having the sidebrowser as a separate app - so only people interested on this specific feature would install it. (especially for Windows and Linux! It's nice discovering there's an app like Slidepad for Mac, but I would love a FOSS version of it)
a. It can be developed or maintained by the same people (Zen team) if they have enough energy for it, or 
b. it could be developed by other developers based on Firefox*, so we can still get it synced to Mozilla account (for extensions and passwords support) - [PWAsForFirefox](https://github.com/filips123/PWAsForFirefox) is a good example for this kind of app/project.

(*or even based on Zen?? the devs would need to work with Zen devs to make it compatible with the side panels, though)
(edit: the more I ruminate with this idea, I think the ideal condition would be... to have someone developing the sidebrowser mode initially, and eventually joining the Zen team to help integrate this sidebrowser mode with Zen and help Mauro developing Zen too)

**Reply by rp1231** [reply]
An optional keyboard shorcut field to hide/show the browser seems like a manageable task.
But I'm just a beginner at programming so I wouldn't know the amount of work it would actually take to implement this feature.

**Reply by rp1231** [reply]
More than happy to wait for the feature to be released.
Its a feature request, not a feature demand :)
